react/src/Navigation.tsx

```diff
@@ -189,6 +191,12 @@ const routes: RouteItem[] = [
         icon: <SupervisedUserCircle />,
         requiresAdmin: true,
     },
+    {
+        pageName: "Search Variants",
+        path: "/search-variants",
+        main: SearchVariants,
+        icon: <SearchIcon />,
+    },
```

Comment:
Let's move this below Analyses in the sidebar and shorten the label to `Variants`. Shorten the path to match too.

Reply:
updated in [e75eaec](https://github.com/ccmbioinfo/stager/commit/e75eaec8e72fd65b311c9be77b5887c0a58c4988)